Content.Server/VoiceMask/VoiceMaskSystem.cs

```diff
@@ -50,6 +59,18 @@ private void OnChangeVerb(Entity<VoiceMaskComponent> entity, ref VoiceMaskChange
         UpdateUI(entity);
     }
 
+    private void OnChangeSound(Entity<VoiceMaskComponent> entity, ref VoiceMaskChangeSoundMessage msg)
+    {
+        if (msg.Sound is { } id && !_proto.HasIndex<SpeechSoundsPrototype>(id))
```

Comment:
I'd probably just manually check for null for clarity rather than doing the null pattern matching syntax.

```suggestion
        if (msg.Sound is not null && !_proto.HasIndex<SpeechSoundsPrototype>(msg.Sound))
```